### PR TITLE
fix: upgrade electron examples

### PR DESCRIPTION
**Pull request by hugomrdias**
upgrade to electron 5 
remove rebuild, apparently there is no need for it

### CODE REVIEW

**Comment by alanshaw** [comment]
Can rebuild.sh be removed now?

**Comment by alanshaw** [comment]
Can the [README](https://github.com/ipfs/js-ipfs#can-i-use-ipfs-in-my-electron-app) be updated to remove the rebuild script? We should probably also link to the example here too.

**Comment by hugomrdias** [comment]
PR updated and https://github.com/ipfs/notes/issues/256 also updated

File: README.md [hunk 1]
```diff
@@ -946,25 +946,8 @@ A way to mitigate this in Chrome, is to run your IPFS node inside a Service Work
...
-export npm_config_disturl=https://atom.io/download/electron
-# Tell node-pre-gyp that we are building for Electron.
-export npm_config_runtime=electron
-# Tell node-pre-gyp to build module from source code.
-export npm_config_build_from_source=true
-# Install all dependencies, and store cache to ~/.electron-gyp.
-HOME=~/.electron-gyp npm install
-```
-
-If you find any other issue, please check the [`Electron Support` issue](https://github.com/ipfs/js-ipfs/issues/843).
```
Comment (daviddias) [inline README.md]: RAD!

**Pull request merged**